source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -369,6 +369,30 @@ progress.
 For more information on DDL operations, see
 :ref:`txn-prod-considerations-ddl`. 
 
+Network Latency
+~~~~~~~~~~~~~~~
+
+Network latency or long physical distances between migration components
+can negatively affect sync speed.
+
+Latency between ``mongosync`` and destination shards
+  For each operation on the source cluster, ``mongosync`` does two 
+  roundtrips to the destination server. The larger the latency, the 
+  slower the sync.
+
+Latency between destination shards
+  Mongosync writes operations and updates its own metadata on the 
+  destination cluster in batches in a transaction. This can result in 
```

Comment:
```suggestion
  Mongosync does operations and updates its own metadata in batches in a transaction on the 
  destination cluster. This can result in 
```

Reply:
Updated. Thanks!